tests/cli/test_cli_static.py

```diff
@@ -0,0 +1,69 @@
+import os
+from unittest.mock import patch
+
+import pytest
+import yaml
+from click.testing import CliRunner
+
+from bigquery_etl.static import publish
+
+VALID_WORKGROUP_ACCESS = [
+    dict(
+        role="roles/bigquery.dataViewer",
+        members=["workgroup:mozilla-confidential"],
+    )
+]
+
+TABLE_METADATA_CONF = {
+    "friendly_name": "test",
+    "description": "test",
+    "owners": ["test@example.org"],
+    "workgroup_access": VALID_WORKGROUP_ACCESS,
+}
+
+DATASET_METADATA_CONF = {
+    "friendly_name": "test",
+    "description": "test",
+    "dataset_base_acl": "derived",
+    "workgroup_access": VALID_WORKGROUP_ACCESS,
+}
+
+
+class TestStatic:
+
+    @pytest.fixture
+    def runner(self):
+        return CliRunner()
+
+    @patch("google.cloud.bigquery.Client.load_table_from_file")
+    @patch("google.cloud.bigquery.Client.get_table")
+    @patch("google.cloud.bigquery.Client.update_table")
```

Comment:
Does patching just client work?  Just in case other client calls are added.  I don't remember the exact behaviour but you might need to do `@patch("bigquery_etl.static.bigquery.Client")`

I just looked at the code and there's a call to the deprecated `client.dataset` which would need to be replaced by `DatasetReference(project, dataset_id)`